sql/src/main/java/io/crate/analyze/InsertFromValuesAnalyzer.java

```diff
@@ -274,7 +280,7 @@ private void addValues(DocTableRelation tableRelation,
             final ColumnIdent columnIdent = column.ident().columnIdent();
             Object value;
             try {
-                valuesSymbol = valueNormalizer.normalizeInputForReference(valuesSymbol, column);
+                valuesSymbol = valueNormalizer.normalizeInputForReference(valuesSymbol, column, user);
```

Comment:
I think the valueNormalizer shouldn't require the user. All it does is retrieve References from the tableInfo - it currently re-retrieves the tableInfo but I think we could change that so that the tableInfo is passed into it as argument since the tableInfo is available here